src/marketplace.css

```diff
@@ -177,8 +177,8 @@
     margin: 0;
 
     /* Padding is only for <a> elements where button styles are applied,
-   buttons elements should have zero padding */
-    padding: 20px 0 0 0;
+		buttons elements should have zero padding */
+    padding: 16px 0 0 0;
```

Comment:
This is likely to break buttons and baselines on every page.
Let's not do it on marketplace.css level!
`SectionHero.css` is better less scary place to make this change.